### PR TITLE
Release 64.0.0

### PR DESCRIPTION
**Pull request by mcmire**
See changelogs.

### CODE REVIEW

File: packages/gas-fee-controller/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.0.1]
+### Changed
+- Add dependencies `eth-query` and `babel-runtime` ([#1447](https://github.com/MetaMask/core/pull/1447))
+-
+### Fixed
+- Fix bug where `query` function failed to call built-in EthQuery methods ([#1447](https://github.com/MetaMask/core/pull/1447))
```
Comment (Gudahtt) [inline packages/gas-fee-controller/CHANGELOG.md:14]: This bug didn't relate to this controller. That PR just improved the type of an internal method, effectively having no impact on the package beyond adding a dependency

Comment (Gudahtt) [inline packages/gas-fee-controller/CHANGELOG.md:14]: Here's a suggestion on addressing the other comment I left:

```suggestion
  - These dependencies were added to improve internal types. This has no impact on users of the package.
```

Comment (Gudahtt) [reply packages/gas-fee-controller/CHANGELOG.md:14]: Oh good catch, thanks, yes that sounds good

Comment (mcmire) [reply packages/gas-fee-controller/CHANGELOG.md]: Okay, updated here: [dc603d9](https://github.com/MetaMask/core/pull/1470/commits/dc603d900afd2492f12aa148cd4a8d8c9a5c7d2c)

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**